src/main/scala/org/broadinstitute/dsde/firecloud/model/package.scala

```diff
@@ -1,43 +1,31 @@
 package org.broadinstitute.dsde.firecloud
 
-import akka.http.scaladsl.model.{StatusCode => AkkaStatusCode}
-import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import akka.http.scaladsl.model.{ContentTypes, HttpEntity, HttpResponse, StatusCodes}
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{MalformedRequestContentRejection, MethodRejection, RejectionHandler}
 import org.broadinstitute.dsde.rawls.model.{ErrorReport, ErrorReportSource}
-import spray.http.MediaTypes.`text/plain`
-import spray.http.{ContentType, HttpEntity, HttpResponse, StatusCode => SprayStatusCode, StatusCodes}
-import spray.routing.{MalformedRequestContentRejection, RejectionHandler}
-import spray.client.pipelining._
-import spray.httpx.SprayJsonSupport._
-import scala.language.implicitConversions
 
-import scala.util.Try
+import scala.language.implicitConversions
 
 package object model {
   implicit val errorReportSource = ErrorReportSource("FireCloud")
 
-  implicit def spray2akkaStatus(spray: SprayStatusCode): AkkaStatusCode = {
-    AkkaStatusCode.int2StatusCode(spray.intValue)
-  }
+  import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport._
+  import spray.json._
 
-  implicit def akka2sprayStatus(akka: AkkaStatusCode): SprayStatusCode = {
-    SprayStatusCode.int2StatusCode(akka.intValue)
-  }
+  implicit val defaultErrorReportRejectionHandler = RejectionHandler.default.mapRejectionResponse {
```

Comment:
i believe these rejection handlers are correctly reimplemented but i'm not certain that both of them get properly utilized. TBD.

Reply:
I tested and refactored these - I am confident they are (it is) working now.